general_rules/Organization.tex

```diff
@@ -128,13 +128,10 @@ \subsection{Scoring System}
 \label{rule:score_system}
 
 Each task has a main objective and a set of bonus scores.
-To score in a test, a team must successfully accomplish the main objective of the task; bonuses are not awarded otherwise.
+To score in a test, a team must successfully accomplish a main objective of the task; bonuses are not awarded otherwise.
```

Comment:
So what does this mean exactly? a team must score more than 0 points on the main objective? Then we should formulate is as such.